backends/gstreamer/render-android/lib.rs

```diff
@@ -192,42 +188,37 @@ impl Render for RenderAndroid {
             ));
         }
 
-        let vsinkbin = gst::ElementFactory::make("glsinkbin", Some("servo-media-vsink"))
-            .map_err(|_| PlayerError::Backend("glupload creation failed".to_owned()))?;
-
         let caps = gst::Caps::builder("video/x-raw")
             .features(&[&gst_gl::CAPS_FEATURE_MEMORY_GL_MEMORY])
             .field("format", &gst_video::VideoFormat::Rgba.to_string())
             .field("texture-target", &gst::List::new(&[&"2D", &"external-oes"]))
             .build();
-        appsink
-            .set_property("caps", &caps)
-            .expect("appsink doesn't have expected 'caps' property");
+        appsink.set_property("caps", &caps);
```

Comment:
```suggestion
        appsink.set_caps(&caps);
```

?

Reply:
I cannot do that in this context, as the `appsink` here actually is a `gst::Element`. Probably we could do that by changing the code a bit more to support `T: glib::IsA<gst_base::BaseSink>` or something, but seems like a bigger refactor.

Reply:
You could `downcast_ref::<gst_app::AppSink>()` / `downcast::<gst_app::AppSink>()` somewhere. The code seems to assume that this is an `appsink` so that assumption should probably be made explicit as early as possible.